### PR TITLE
Introduction, Security Changes

### PR DESCRIPTION
**Pull request by dudeso**
I made some improvements to the Introduction to clarify (the lack of) hardware prerequisites. 

Moved some pieces from "Before..." into the security section

### CODE REVIEW

File: sccm/protect/deploy-use/use-device-guard-with-configuration-manager.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ manager: angrobe
 ## Introduction
 Windows Device Guard is a group of Windows 10 features that are designed to protect PCs against malware and other untrusted software. It prevents malicious code from running by ensuring that only approved code, that you know, can be run.
 
+Device Guard encompasses both software and hardware based security functionality. Configurable code integrity is a software based security layer that enforces an explicit list of software that is allowed to run on a PC. On its own, configurable code integrity does not have any hardware or firmware prerequisites. Device Guard policies deployed with Configuration Manager enable a configurable code integrity policy on PCs in targeted collections that meet the minimum Windows version and SKU requirements outlined below. Optionally, hypervisor based protection of code integrity policies deployed through Configuration Manager can be enabled through Group Policy on capable hardware.
```
Comment (dudeso) [inline sccm/protect/deploy-use/use-device-guard-with-configuration-manager.md]: Added to clarify DG flavors and hardware requirements

File: sccm/protect/deploy-use/use-device-guard-with-configuration-manager.md [hunk 2]
```diff
@@ -63,10 +65,9 @@ When you deploy a policy, typically, the following executables will be allowed t
 
 Before you configure or deploy Device Guard policies, read the following information:
 
-- To use Device Guard, PCs you manage must be running the Windows 10 Creators Update, or later.
```
Comment (dudeso) [inline sccm/protect/deploy-use/use-device-guard-with-configuration-manager.md]: Moved some pieces to the security section

**Comment by robstackmsft** [comment]
@dudeso Changes look great. Thanks!

**Comment by robstackmsft** [comment]
#sign-off

**Comment by robstackmsft** [comment]
@mtillman

**Pull request merged**